### DISCUSSION TITLE
Torch-TensorRT

### DISCUSSION THREAD

**Discussion by Jonathan-v-S** [neonbjb/tortoise-tts#653] [Ideas]
Can Torch-TenorRT improve performance of Tortoise-TTs?
https://developer.nvidia.com/blog/accelerating-inference-up-to-6x-faster-in-pytorch-with-torch-tensorrt/

**Comment by manmay-nakhashi** [comment]
@CodeHeroJo it can feel free to raise a PR if you can implement that.

**Comment by C00reNUT** [comment]
It certainly can https://developer.nvidia.com/blog/speeding-up-text-to-speech-diffusion-models-by-distillation/

**Reply by CoderCowMoo** [reply]
This isn't the use of Torch-TensorRT though, this is distillation of the DDPM.
